src/server/views/lists/partials/lawyers/macros.njk

```diff
@@ -13,27 +32,27 @@
     </p>
   {% endif %}
   <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">
-    <strong>Address</strong>: 
-      {{ item.address.firstLine }} 
+    <strong>Address</strong>:
+      {{ item.address.firstLine }}
       {{ "-" if item.address.secondLine }} {{ item.address.secondLine }}
-      {{ "-" if item.address.postCode }} {{ item.address.postCode }} 
+      {{ "-" if item.address.postCode }} {{ item.address.postCode }}
       {{ "-" if item.address.city }} {{ item.address.city }}
   </p>
-  {% if item.jsonData.email %}
+  {% if item.jsonData.emailAddress %}
     <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">
-      <strong>Email</strong>: <a class="govuk-link" href="mailto:{{ item.jsonData.email }}">{{ item.jsonData.email }}</a>
+      <strong>Email</strong>: <a class="govuk-link" href="mailto:{{ item.jsonData.emailAddress }}">{{ item.jsonData.emailAddress }}</a>
     </p>
   {% endif %}
-  {% if item.jsonData.website %}
+  {% if item.jsonData.websiteAddress %}
     <p class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1">
-      <strong>Website</strong>: <a class="govuk-link" href="{{ enforceHttps(item.jsonData.website) }}" rel="noopener noreferrer" target="_blank">{{ item.jsonData.website }}</a>
+      <strong>Website</strong>: <a class="govuk-link" href="{{ item.jsonData.websiteAddress }}" rel="noopener noreferrer" target="_blank">{{ item.jsonData.websiteAddress }}</a>
     </p>
   {% endif %}
 
-  {% if (not item.jsonData.email) or (not item.jsonData.website) %}
+  {% if (not item.jsonData.emailAddress) or (not item.jsonData.websiteAddress) %}
```

Comment:
Any reason for appending Address to website and email?  I feel the verbosity doesn't really add much and is more like noise to read through.

Reply:
It was what I got with the exported form JSON so I left as is

Reply:
@alisalaman-cyb emails are what get sent to somebody's inbox. adding `address` makes it clearer as to what we're referring to... (@superafroman's pet peeve 😂)